### PR TITLE
Execute flow from frontend

### PR DESCRIPTION
**Pull request by orhanrauf**
Basic functionalities to do so are now there. The front-end is not pretty, but sufficient to close this branch. Will work on separate improvements in other branches.

### CODE REVIEW

File: backend/app/main.py [hunk 1]
```diff
@@ -77,6 +77,8 @@ async def exception_logging_middleware(request: Request, call_next):
         return response
     except Exception as exc:
         # Log the exception with traceback
+        
+        print(f"Unhandled exception: {exc}\n{traceback.format_exc()}")
```
Comment (lennertjansen) [inline backend/app/main.py:81]: is it necessary to print this exception if you're already logging it?

File: backend/app/models/flow_run.py [hunk 2]
```diff
@@ -19,6 +23,16 @@ class FlowRun(Base):
...
+    created_by_email = mapped_column(String, ForeignKey("user.email"), nullable=False)
+    modified_by_email = mapped_column(String, ForeignKey("user.email"), nullable=False)
+
+    created_date: Mapped[datetime] = mapped_column(DateTime(timezone=True), server_default=func.now(), nullable=False)
+    modified_date: Mapped[datetime] = mapped_column(DateTime(timezone=True), server_default=func.now(), server_onupdate=func.now(), nullable=False)
 
     task_runs = relationship("TaskRun", back_populates="belongs_to_flow_run", cascade="all, delete-orphan")
     belongs_to_flow = relationship("Flow", back_populates="flow_runs")
+    
+    created_by: Mapped["User"] = relationship(back_populates="created_flow_runs", foreign_keys="FlowRun.created_by_email")
```
Comment (lennertjansen) [inline backend/app/models/flow_run.py:36]: Are both fields `created_date` and `created_by` necesarry? What's the difference between them?

Comment (orhanrauf) [reply backend/app/models/flow_run.py:36]: created_by = user
created_date = datetime

File: backend/app/core/task_preparation_generator.py [hunk 3]
```diff
@@ -5,12 +5,13 @@
 from app.core.config import settings
 from app.core.logging import logger
 from app.db.session import SessionLocal
-from app.schemas.task_prep_prompt import TaskPrepPromptBase
+from app.schemas.flow_request import FlowRequest
```
[the remark this answers is not in the document]
Comment (orhanrauf) [reply backend/app/core/task_preparation_generator.py:8]: FlowRequest is the universal way of calling the the user's request for a flow generation

File: backend/app/models/flow_run.py (hunk 2 above)
Comment (lennertjansen) [reply backend/app/models/flow_run.py:36]: Sorry, I seem to have misread the field-name. I thought it said `created_on` instead of `created_by`.

**Pull request merged**